packages/react-components/src/components/gauge/gaugeBase.tsx

```diff
@@ -48,8 +48,8 @@ export const GaugeBase: React.FC<GaugeBaseProperties> = ({
     chartRef,
     {
       isLoading,
-      thresholds,
-      gaugeValue,
+      thresholds: error ? [] : thresholds,
```

Comment:
Do we wanna break the whole widget if there's an error? I think we should still display whatever values we have and show some error state on top of it if there is any error